SwiftLeeds/Views/Sponsors/SponsorsView.swift

```diff
@@ -93,13 +95,35 @@ private extension SponsorsView {
                 }
             }.foregroundColor(.clear)
     }
+    
+    func jobtile(for sponsor: Sponsor) -> some View {
```

Comment:
Can this be private?

Reply:
<img width="599" alt="image" src="https://github.com/SwiftLeeds/swiftleeds-ios/assets/7802406/21962a61-7e6a-4640-8f9b-67f468859253">

this is already in a private extension only, correct me if I misunderstood

Reply:
Wasn't obvious from the code shown 👍